mne/viz/evoked.py

```diff
@@ -1024,6 +1024,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:
+        evoked.pick_types(eeg=True, eog=True, seeg=True, meg=True)
```

Comment:
Ok, can you open an issue so that other may join the discussion?

Although I agree with you that EOG can be treated like EEG in many cases, we should apply the rule of least surprise in the API. We could mention/discuss this kind of issue in the tutorials.